### PR TITLE
prep-html-script-tag-Carolyne Golias

### PR DESCRIPTION
**Pull request by GCarolyne**

### CODE REVIEW

**Comment by GCarolyne** [comment]
Screenshot 2024-09-29 at 5 26 46 PM

File: prep-html-script-tag/index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  <head>
+    <title>HTML Script Tag</title>
+    <body>
+      <script>
+        console.log('I have added JS to a script tag in the HTML!');
+      </script>
+
+      <script src="main.js">
+        console.log(' I have added a script file via the src attribute ');
+      </script>
```
Comment (shawnkost) [inline prep-html-script-tag/index.html]: For this one, you should not have anything in-between the opening and closing `script` tag. You'll just load in the `main.js` file and your `console.log` will just be in there

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Carolyne!

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**